.azure-pipelines/client.yml

```diff
@@ -0,0 +1,180 @@
+trigger:
+- psSdkJson6
+
+#Variables: 
+#  'source.scope' list of projects to build (Ex: SDKs\KeyVault\dataPlane)
+#  'build.useLatest' indicates whether or not to use build bootstrap files from the LTSBuildTools branch
+#  'sign.pattern' the pattern used to find assemblies for SN and AuthentiCode signing
+#  'system.debug' bool for debug tracing
+
+#TODO: sign.pattern needs better filtering (not KeyVault specific, exclude test assemblies).
+
+variables:
+  source.scope: 'SDKs\KeyVault\dataPlane'
+  build.useLatest: 'false'
```

Comment:
NIT: @kurtzeborn you want this to be true.
Once the changes are merged to the main build branch, at that point you want to make this as false.

Reply:
Agreed.  The variable definitions are being removed.